pkg/controller/user/user_controller.go

```diff
@@ -83,11 +96,32 @@ func (r *ReconcileUser) Reconcile(request reconcile.Request) (reconcile.Result,
 	return reconcile.Result{}, err
 }
 
-func mapUserNames(users *usersv1.UserList) []string {
+func mapUserNames(users *usersv1.UserList, groups *usersv1.GroupList) []string {
 	var result = []string{}
 	for _, user := range users.Items {
-		result = append(result, user.Name)
+		// Certain users such as sre do not need to be added
+		if !userInExclusionGroup(user, groups) {
+			result = append(result, user.Name)
+		}
 	}
 
 	return result
 }
+
+func userInExclusionGroup(user usersv1.User, groups *usersv1.GroupList) bool {
+
+	// Below is a slightly complex way to determine if the user exists in an exlcusion group
+	// Ideally we would use the user.Groups field but this does not seem to get populated.
+	for _, group := range groups.Items {
+		for _, xGroup := range exclusionGroups {
+			if group.Name == xGroup {
+				for _, groupUser := range group.Users {
+					if groupUser == user.Name {
+						return true
+					}
+				}
+			}
+		}
+	}
+	return false
+}
```

Comment:
Looks to be comon function in here and rhsso reconciler, what do you think on moving this to `userHelper.go`?